index.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const bcrypt = require('bcryptjs');
+const db = require('./data/db');
+const jwt = require('jsonwebtoken');
+const server = express();
+
+server.use(express.json());
+
+server.post('/api/register', (req, res) => {
+  const user = req.body;
+  const hash = bcrypt.hashSync(user.password, 14);
+  user.password = hash;
+
+  db('users')
+    .insert(user)
+    .then((ids) => {
+      db('users')
+        .where({ id: ids[0] })
+        .first()
+        .then((user) => {
+          const token = generateToken(user);
+          res.status(201).json(token);
+        });
+    })
+    .catch((error) => {
+      res.status(500).json({ error });
+    });
+});
+
+const secret = 'I love icecream!';
```

Comment:
I know for a fact that this is not a secret...  👍

Reply:
😂

Reply:
🍨